.github/workflows/test.yml

```
@@ -22,3 +26,113 @@ jobs:
           cache: 'npm'
       - run: npm ci
       - run: npm run lint
+
+  build-windows-bundle-and-test:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+
+      - name: Use Node.js v14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16'
+
+      - name: Set up Python 3.9
```

Don't think we should bother including the version in the title, unless we set up different versions explicitly.

```suggestion
      - name: Set up Python
```